src/store/util.ts

```diff
@@ -0,0 +1,6 @@
+function buildActionOrMutationName( namespace: string, actionName: string ): string {
```

Comment:
This could be more generic, in reason of there could be n-namespaces. The heads are always namespaces and the tail are the action.

Reply:
Good idea! Done.